src/app/personside/visittkort/body/adresse/Adresse.tsx

```diff
@@ -153,7 +155,16 @@ function hentPeriode(periode?: Periode) {
         const fra = formaterDato(periode.fra);
         const til = formaterDato(periode.til);
         return (
-            <Undertekst>{fra} - {til}</Undertekst>
+            <EtikettMini>{fra} - {til}</EtikettMini>
```

Comment:
usikker på hva som skal brukes her, men kjør på med etikettMini for øyeblikket. Men kanskje noe forklarende tekst hadde gjort seg..? f.eks `gyldig {fra}-{til}`